maps/unused/mushroom_new_walls.dmm

```diff
@@ -27016,12 +27016,8 @@
 	},
 /area/station/bridge)
 "nWp" = (
-/obj/item/device/camera_viewer{
-	network = "Zeta"
-	},
-/obj/item/device/camera_viewer{
-	network = "Zeta"
-	},
+/obj/item/device/camera_viewer,
+/obj/item/device/camera_viewer,
```

Comment:
these are for telesci right?

```suggestion
/obj/item/device/camera_viewer/telesci,
/obj/item/device/camera_viewer/telesci,
```

Reply:
probably, I just used vsc to manually edit, making change

Reply:
yea def are from map diff